uk.co.compucorp.civicrm.hrcore/js/src/hrcore/helpers/help-icon-injector.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env amd */
+
+(function (CRM) {
+  define(function () {
+    'use strict';
+
+    return injectHelpIcon;
+
+    /**
+     * Injects a help icon to a form field
```

Comment:
it's not precise i think, aren't you appending an help icon to a form field?

Reply:
Done.